### PR TITLE
test use_hotkeys_ref! via demo

### PR DESCRIPTION
**Pull request by friendlymatthew**
@JustBobinAround 

I created a div in `scope_a` where if you click into it and press `k` it should update an `is_green` state which is used to conditionally display the background color of the div with the `node_ref`. 

However this behavior is not working as expected. Could you take a look at this and see?

### CODE REVIEW

File: example/demo/src/app.rs [hunk 1]
```diff
@@ -106,19 +106,13 @@ fn HomePage() -> impl IntoView {
...
 
     let a_ref = use_hotkeys_ref!(("k", "scope_a") => move |_| {
-        current_ref.set("ref_a");
-        set_count.update(|count| {
-            *count += 1;
-        })
+        if is_green.get() {
+            is_green.set(false)
+        } else {
+            is_green.set(true)
```
Comment (friendlymatthew) [inline example/demo/src/app.rs:112]: specifically here

File: example/demo/src/app.rs [hunk 2]
```diff
@@ -179,17 +166,23 @@ fn HomePage() -> impl IntoView {
...
                                     </div>
                                 </div>
+                                <div
+                                    _ref=a_ref
+                                    class:green=move || is_green.get() == true
+                                    class:yellow=move || is_green.get() == false
+                                >
+                                    <p>click on me and press spacebar</p>
+                                </div>
                             </div>
```
Comment (friendlymatthew) [inline example/demo/src/app.rs:176]: disregard the `spacebar`. it should be `k`

File: example/demo/src/app.rs [hunk 3]
```diff
@@ -147,30 +141,23 @@ fn HomePage() -> impl IntoView {
...
+                        </div>
                     </div>
                     <div class="flex-1 grid grid-col-1 lg:grid-cols-2">
                         <div
                             id="scope_a"
                             class:active=move || current_scope.get() == "scope_a"
                             class:not-active=move || current_scope.get() != "scope_a"
-                            _ref={a_ref} 
-                            tabIndex={-1}
+                            tabIndex=-1
```
Comment (JustBobinAround) [inline example/demo/src/app.rs:157]: notice how `tabIndex=-1` one is here. This should be in the div where `_ref` attribute is...

File: example/demo/src/app.rs [hunk 4]
```diff
@@ -179,17 +166,23 @@ fn HomePage() -> impl IntoView {
                                         <p>"press 'Escape' to reset the count"</p>
                                     </div>
                                 </div>
+                                <div
+                                    _ref=a_ref
+                                    class:green=move || is_green.get() == true
+                                    class:yellow=move || is_green.get() == false
```
Comment (JustBobinAround) [inline example/demo/src/app.rs:172]: you need to have `tabIndex=-1` for `_ref` attribute to be clickable/work... take a look at: https://react-hotkeys-hook.vercel.app/docs/documentation/useHotkeys/scoping-hotkeys

Comment (friendlymatthew) [reply example/demo/src/app.rs:172]: Gotcha -- I'll make sure to note this in our Readme

**Pull request merged**